mlflow/environment_variables.py

```diff
@@ -202,3 +202,9 @@ def get(self):
 MLFLOW_DEFAULT_PREDICTION_DEVICE = _EnvironmentVariable(
     "MLFLOW_DEFAULT_PREDICTION_DEVICE", str, None
 )
+
+#: Specifies whether or not to allow using a file URI as a model version source.
+# (default: ``False``).
```

Comment:
Can we add some documentation indicating that this is potentially risky because it can allow access to arbitrary files on the specified filesystem and should be used with great caution?

Reply:
Updated the comment!